nss-day-cohort-45/tabloidfullstack-giant-gibbons
# Cjd mergemagic

## Categories, Post Edit and Post Delete

###To test:
Log in
1. Check that on the post page, the category shows in place of the content.
2. From Posts or My Posts, check that you can Edit or Delete ONLY YOUR OWN POSTS.
3. From a Post Detail, see that there is a "Comment" button that takes you to the post's comments (there may not be any).
